starlette/responses.py

```diff
@@ -33,25 +33,31 @@ def guess_type(
 class Response:
     media_type = None
     charset = "utf-8"
+    default_status_code = 200
+    default_response = b""
```

Comment:
Let's not have these attributes - it's not a bit of behaviour that we'd want subclasses to modify.

Reply:
You're right, the `default_status_code` should be removed. But need the `default_response` to be modified by `JSONResponse`. Maybe that indicates an issue here but not calling render on json will create a non-valid jSON response b"".